Python/Discovering Subdomains/README.md

```diff
@@ -0,0 +1,32 @@
+# <b>Discovering Subdomains</b>
+
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+## Discovering Subdomains : 🚀
+
+Searching for subdomains manually would take forever. Luckily, we don't have to do that, in this script, 
+I will build a subdomain scanner in Python using requests library. 
+
+## Discovering Subdomains Instructions: 👨🏻‍💻
+
+### Step 1:
+
+    Open Termnial 💻
+    
+### Step 2:
+
+   Install the required packages  that is in the `requirement.txt`
+   
+### Step 3:
+
+    Locate to the directory where python file is located 📂
+
+### Step 4:
+    
+    Enter the domaion of your choice like `Email` `blog` etc this was depend on you And then run the `program.py` file
+
+### Step 5:
+
+    Sit back and Relax. Let the Script do the Job. ☕
+    
```

Comment:
Please add link to `Screenshots` here and brief explanation of what the script actually does for anyone using it.